cratedb_django/schema.py

```diff
@@ -68,3 +71,28 @@ def alter_field(self, model, old_field, new_field, strict=False):
             # You cannot change types after table creation.
             return
         return super().alter_field(model, old_field, new_field, strict)
+
+    def table_sql(self, model):
+        sql = list(super().table_sql(model))
+
+        partition_by = getattr(model._meta, "partition_by", OMITTED)
+        if partition_by is not OMITTED:
+            if not isinstance(partition_by, Sequence) or not partition_by:
+                raise ValueError(
+                    "partition_by has to be a non-empty sequence, " "e.g. ['id']"
```

Comment:
Just a formatting nit.
```suggestion
                    "partition_by has to be a non-empty sequence, e.g. ['id']"
```